taskcluster_proxy.go

```diff
@@ -61,7 +65,7 @@ func (l *TaskclusterProxyTask) Start() *CommandExecutionError {
 			Certificate: l.task.TaskClaimResponse.Credentials.Certificate,
 			ClientID:    l.task.TaskClaimResponse.Credentials.ClientID,
 		},
-		l.task.TaskID,
+		scopes,
```

Comment:
They aren't actually authorizedScopes in this case -- I noticed it was done that way, but that's really just "cheating" and keeping the list of scopes in an otherwise-unused field.

Reply:
They really are AuthorizedScopes - see [how they are interpreted on the other end](https://github.com/taskcluster/taskcluster-proxy/blob/9bb2082b1098b3e9c0e45910e6ceeef96016e10a/main.go#L143-L145).